serverlessworkflow/modules/ROOT/pages/eventing/working-with-callbacks.adoc

```diff
@@ -61,9 +80,9 @@ Then it declares a Callback state that will publish an event of type `resume` an
     }
 ----
 
-The event of type `resume` is consumed by a link:{kogito_sw_examples_url}/serverless-workflow-callback-quarkus/src/main/java/org/kie/kogito/examples/PrintService.java[event listener]. This event listener publishes a new event of type `wait`.
+In the previous example, an link:{kogito_sw_examples_url}/serverless-workflow-callback-quarkus/src/main/java/org/kie/kogito/examples/PrintService.java[event listener] consumes the `resume` type CloudEvent and publishes a new `wait` type CloudEvent. Following is an example of a Java method that publishes the `wait` type CloudEvent:
```

Comment:
Here we change from passive to non passive (an example of whay I said in the previous comment)

Reply:
BTW, when moving this to active, you need to introduce "in the previous example" which is confusing because the previous example is not at all related with this parragraph, that introduce a new convept, the java event listener that consumes the event

Reply:
Done, changed the sentence as follows:

An link:{kogito_sw_examples_url}/serverless-workflow-callback-quarkus/src/main/java/org/kie/kogito/examples/PrintService.java[event listener] consumes the `resume` type CloudEvent and publishes a new `wait` type CloudEvent. Following is an example of a Java method that publishes the `wait` type CloudEvent: